mne/dipole.py

```diff
@@ -1055,9 +1048,8 @@ def fit_dipole(evoked, cov, bem, trans=None, min_dist=5., n_jobs=1,
         The dipole fits. A :class:`mne.DipoleFixed` is returned if
         ``pos`` and ``ori`` are both not None, otherwise a
         :class:`mne.Dipole` is returned.
-    residual : ndarray, shape (n_meeg_channels, n_times)
-        The good M-EEG data channels with the fitted dipolar activity
-        removed.
+    residual : instance of EvokedArray
```

Comment:
it actually returns so far a copy of the passed evoked so it's not always an EvokedArray